components/utils/__tests__/useDomDimension.test.tsx

```diff
@@ -0,0 +1,14 @@
+import useDOMDimension from '../use-dom-dimension'
+import React from 'react'
+import { mount } from 'enzyme'
+describe('useFixedWidth', () => {
+  it('should work on a null component', () => {
+    function App() {
+      //todo jest can not get width value correctly
+      const ref = useDOMDimension<HTMLDivElement>('offsetWidth')[1]
+      return <div>{false && <div style={{ width: 100 }} ref={ref}></div>}</div>
```

Comment:
Expect a `expect` here. Why there's a `false &&`

Reply:
cover the case component did not mount which cause the `ref` to be `null`